platform-hub-api/Dockerfile

```diff
@@ -1,22 +1,40 @@
-FROM ruby:2.3.3
-
-RUN apt-get update -y && apt-get -u upgrade -y
-RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ jessie-pgdg main" >> /etc/apt/sources.list.d/pgdg.list \
-    && wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - \
-    && apt-get update && apt-get install -y libpq-dev
-
-WORKDIR /app
+FROM quay.io/ukhomeofficedigital/centos-base
 
+ENV RUBY_VERSION 2.3.3
 ENV RAILS_ENV production
 ENV RACK_ENV production
 
-COPY Gemfile Gemfile.lock ./
-RUN bundle install --jobs 20 --retry 5 --deployment --without development test --no-cache
+# Update packages & Install RVM
+RUN yum update -y \
+    && yum install -y \
+       which gcc-c++ patch readline readline-devel zlib zlib-devel \
+       libyaml-devel libffi-devel openssl-devel make bzip2 autoconf \
+       automake libtool bison iconv-devel sqlite-devel nodejs \
+    && curl -sSL https://rvm.io/mpapis.asc | gpg --import - \
+    && curl -L get.rvm.io | bash -s stable \
+    && yum clean all
+RUN /bin/bash -l -c "rvm requirements"
+RUN /bin/bash -l -c "rvm install 2.3.3"
+RUN /bin/bash -l -c "rvm use 2.3.3 --default"
+RUN /bin/bash -l -c "gem install bundler --no-ri --no-rdoc"
 
-COPY . .
+# Install postgresql for libmq dependency
+RUN yum install -y postgresql-devel postgresql-libs postgresql-contrib
 
+# Create app user
+WORKDIR /app
 RUN groupadd -r app && useradd -r -g app app
 RUN chown -R app:app /app
```

Comment:
This one is still required, creates and sets the working directory for the app files to be copied to.